### DISCUSSION TITLE
Ramp case parallel computing issue

### DISCUSSION THREAD

**Discussion by cczz8** [mdolab/dafoam#707] [General]
I am currently learning the Ramp case and would like to use more CPU cores for computation. However, when I changed mpirun --oversubscribe -np 4 python runPrimal.py to mpirun --oversubscribe -np 64 python runPrimal.py in preprocessing.sh, it seems that running ./processing.sh has become slower. Could anyone help me understand why this is happening? Thank you so much for your assistance.

**Comment by friedenhe** [comment]
The ramp case has a very coarse mesh, so using 64 cores is not quite necessary and may actually slow down the simulation.
